components/eamxx/scripts/scripts-tests

```diff
@@ -592,7 +534,7 @@ class TestGatherAllData(TestBaseOuter.TestBase):
 ###############################################################################
 
     CMDS_TO_TEST = [
-        "./gather-all-data './scripts/test-all-scream -m $machine' -l -m $machine",
+        "./gather-all-data 'echo -m $machine' -l -m $machine",
```

Comment:
Nice! Using a simple cmd like echo is much better to test the outer script. No need to add the inner script complexity to the mix.